### PR TITLE
Refactored html

### PR DESCRIPTION
**Pull request by Sharfiik**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <!-- Шапка сайта -->
     <header>Шапка сайта</header>
     <!-- Секция 1 -->
-    <section class="section" id="warranty">
+    <section class="section" id="warranty1">
```
Comment (nagorapro) [inline index.html]: You need to replace the id value with "download"

**Review by ghost — APPROVED** [review]
Done

File: index.html [hunk 2]
```diff
@@ -10,21 +10,21 @@
...
               <img src="./assets/icons/apple.svg" alt="Apple icon">
             </a>
-            <a class="link" href="https://google.com">
+            <a class="download__link" href="https://google.com">
               <img src="./assets/icons/google.svg" alt="Google icon">
             </a>
           </div>
         </div>
-        <div class="image-blokc">
+        <div class="download__image-blokc">
```
Comment (nagorapro) [inline index.html]:
```
<div class="download__image-block">
```

File: index.html [hunk 3]
```diff
@@ -10,21 +10,21 @@
...
-        <div class="text-block">
-          <h1 class="title">Peaceful shopping mindful money</h1>
-          <p class="capy">See how we can help with making your shopping experience and money management more ZEN.</p>
-          <div class="links">
-            <a class="link" href="https://apple.com">
+    <section class="download" id="download"> 
+      <div class="download__wrapper">
+        <div class="download__text-block">
+          <h1 class="download__title">Peaceful shopping mindful money</h1>
+          <p class="download__capy">See how we can help with making your shopping experience and money management more ZEN.</p>
```
Comment (nagorapro) [inline index.html]:
```
<p class="download__copy">
```

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by nagorapro — APPROVED** [review]
Done

**Pull request merged**